projects/fal/src/fal/api.py

```diff
@@ -1099,9 +1101,14 @@ def openapi(self) -> dict[str, Any]:
     def serve(self) -> None:
         import asyncio
 
+        from prometheus_client import Gauge
         from starlette_exporter import handle_metrics
         from uvicorn import Config
 
+        # NOTE: this uses the global prometheus registry
+        app_info = Gauge("fal_app_info", "Fal Application information", ["version"])
```

Comment:
```suggestion
        app_info = Gauge("fal_app_info", "Fal application information", ["version"])
```
not sure about capitalization?